client/src/components/ContentNavigator/const.ts

```diff
@@ -47,7 +47,8 @@ export const Messages = {
   FileOpenError: "The file type is unsupported.",
   FileValidationError: "Invalid file name.",
   FolderDeletionError: "Unable to delete folder.",
-  FolderValidationError: "Invalid folder name.",
+  FolderValidationError:
```

Comment:
I think this language change is probably fine, but we should confirm with @skyeaussie / @Becky-Williamson

Reply:
Yes, this new message matches one of our standard messages and it is acceptable. Thanks

Reply:
thanks!